### PR TITLE
TC10 TC11 TC14

### PR DESCRIPTION
**Pull request by viethuynh140499**

### CODE REVIEW

**Comment by viethuynh140499** [comment]
This PR is testing

**Comment by viethuynh140499** [comment]
TC10 - User can't create account with "Confirm password" is not the same with "Password"

TC11 - User can't create account while password and PID fields are empty

TC14 - User can book 1 ticket at a time

File: as [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+[33m64d25c2[m Merge pull request #2 from viethuynh140499/oklogin
+[33m5ac1072[m "oke login"
+[33m1da01dd[m Merge pull request #1 from viethuynh140499/login
+[33m0de729c[m login with json data
+[33mde01d53[m update
+[33me290a3c[m remote return
+[33mcbab457[m update 3
+[33m9d5020f[m update new
+[33mefeefbd[m update
+[33m2f1da6d[m init
```
Comment (fey031) [inline as:38]: Cái chi kinh ri???

Comment (viethuynh140499) [reply as]: I deleted this file

File: src/main/java/page_objects/RegisterPage.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  public void registerAccount(String username, String password, String confirmPassword, String pid) {
+    this.btnRegister.scrollToView();
+    this.txtEmail.enterText(username);
+    this.txtPassword.enterText(password);
+    this.txtConfirmPassword.enterText(confirmPassword);
+    this.txtPid.enterText(pid);
+  }
+}
```
Comment (tanmle) [inline src/main/java/page_objects/RegisterPage.java:44]: break line.

Comment (viethuynh140499) [reply src/main/java/page_objects/RegisterPage.java]: I fixed it, Thank you

**Comment by viethuynh140499** [comment]
close

**Pull request closed without merging**